### ISSUE TITLE
fftw3 module in M10

### ISSUE THREAD

**Issue by muralidn** [liziscool/cas741_project#47]
@liziscool 

M10 is implemented by fftw3 but there is no information about that module neither in the SRS nor the Design docs (MG, MIG).

![image](https://user-images.githubusercontent.com/71282513/100667007-048b4a80-3380-11eb-9c52-b84e343b5163.png)

**Comment by liziscool** [comment]
Yes, it's a library I am using. I'm not sure where else I should mention that. I was going off the solar water heating example where sometimes it says Implemented by Matlab. I will look into where I need to include more information about it.

**Comment by muralidn** [comment]
if fftw3 is a library and not a general-purpose tool, I think it should be referenced in section 3.1 (System Context) of the SRS.
